ghost/core/test/utils/browser-test-utils.js

```
@@ -0,0 +1,253 @@
+const { JSDOM } = require('jsdom');
```

ESLint

[error] There should be no space after '{'.

(object-curly-spacing)